app/controllers/ApiController.scala

```diff
@@ -0,0 +1,353 @@
+package controllers
+
+import auth.PanDomainAuthentication
+import com.gu.pandomainauth.PublicSettings
+import config.Config
+import play.api.Logging
+import play.api.mvc._
+import play.api.libs.json.Json
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+
+class ApiController (
+  override val controllerComponents: ControllerComponents,
+  val publicSettings: PublicSettings,
+  val config: Config
+) extends BaseController with Logging with PanDomainAuthentication {
+  /**
+   * Create an Action to render an HTML page with a welcome message.
+   * The configuration in the `routes` file means that this method
+   * will be called when the application receives a `GET` request with
+   * a path of `/`.
+   */
+  def index(id: String) = Action {
```

Comment:
I've almost resolved `login.gutools` credentials issues; we can make this an `ApiAuthAction` soon.

There are two PRs outstanding:
- https://github.com/guardian/login.gutools/pull/48
- https://github.com/guardian/janus/pull/2407

Reply:
Awesome. Fingers crossed. 🤞